### DISCUSSION TITLE
Sidedownload error

### DISCUSSION THREAD

**Discussion by Stars-n-S** [aclap-dev/video-downloadhelper#717] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [https @ 000002b02da2ddc0] Stream ends prematurely at 135577, should be 343288\r\n[https @ 000002b02db5a4c0] Stream ends prematurely at 137029, should be 956920\r\n[aac_adtstoasc @ 000002b02d69a640] Error parsing ADTS frame header!\r\n[mp4 @ 000002b02e0f00c0] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 000002b02db81680] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 000002b02da53680] Error muxing a packet\r\n",
    "id": "coapp_failure_95aa6eec-2b7f-48c4-b5f2-33a83476313e",
    "downloadable_id": "downloadable_3353537491124904"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [https @ 000002b02da2ddc0] Stream ends prematurely at 135577, should be 343288\r\n[https @ 000002b02db5a4c0] Stream ends prematurely at 137029, should be 956920\r\n[aac_adtstoasc @ 000002b02d69a640] Error parsing ADTS frame header!\r\n[mp4 @ 000002b02e0f00c0] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 000002b02db81680] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 000002b02da53680] Error muxing a packet\r\n",
    "id": "coapp_failure_95aa6eec-2b7f-48c4-b5f2-33a83476313e",
    "downloadable_id": "downloadable_3353537491124904"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Your problem report says:

> "page_url": "undef"

Normally, that shows the URL of the web page on which you were trying to download a video.  That information is critical to investigating any problem.  You can get around this failure of the error reporting mechanism by posting the URL yourself.  Again, what is needed is the complete URL of a specific web page on which there is a video.

We have hit a number of these ADTS errors recently.  Reloading the web page then trying to get VDH to download the item again often works.  Sometimes it takes several retries, maybe more than 5, more than 10, before it works.  Maybe that will work with your content.  Maybe.  You can't know unless you try.
